docker/README.md

```diff
@@ -43,6 +43,32 @@ Once this is done you should run the following command to build `aarch64` static
 ./docker-static-build.sh aarch64
 ```
 
+### Packages from statically linked binaries
+
+It's possible to create packages (ubuntu/debian & fedora) using already
+built binaries. This means that the build step of the package will be reduced
+to copying the binaries instead of building again from scratch.
+
+Using Ubuntu/Debian and assuming all Tezos binaries are located in the `out` directory:
+```sh
+# DEB_BUILD_OPTIONS=nostrip is needed if the provided binaries have already
+# been stripped of any debug symbols.
+
+docker build -t tezos-ubuntu -f docker/package/Dockerfile-ubuntu .
+docker run -it -e TEZOS_VERSION=v13.0 \
```

Comment:
Does this command suppose to be run from the repo's root from the `docker` directory? For some reason I can't get it to work in both cases :disappointed: